llarp/ev/ev.hpp

```diff
@@ -671,7 +671,8 @@ namespace llarp
 #ifdef _WIN32
 struct llarp_fd_promise
 {
-  void Set(std::pair< int, int >)
+  void
```

Comment:
is `make format` capricious, or did someone forget to run it at some point?

Reply:
`make format` should use `clang-format-8` as 9 and higher makes the source get fugged some how.